testdata/book/cookie.yml

```diff
@@ -0,0 +1,36 @@
+desc: Inherit cookies Example
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    openapi3: https://tryapisproxy.com/spec/httpbin
+vars:
+  cookie:
+    key: "cookie-name"
+    value: "cookie-value"
+debug: ${DEBUG:-true}
+steps:
+  bindUuid:
+    desc: UUIDs are numbered in the image of a session.
+    bind:
+      uuid: faker.UUID()
+  setCookie:
+    desc: Returns a set of response headers from the query string.
+    req:
+      /response-headers?Set-Cookie={{ vars.cookie.key }}%3D{{ uuid }}:
+        get:
+          body: null
+    test: |
+      current.res.status == 200
+      && current.res.cookies[0].Name == vars.cookie.key
+      && current.res.cookies[0].Value == uuid
+  inheritCookie:
+    desc: Returns cookie data.
+    req:
+      /cookies:
+        get:
+          headers:
+            Cookie: "{{ cookies['httpbin.org'][0].Raw }};"
```

Comment:
Hosts change from environment to environment, so we may need the ability to identify them.

#555